systemservices/marketplace/mesg.yml

```diff
@@ -0,0 +1,414 @@
+                    name: ""
+                    description: ""
+                    type: String
+                  manifest:
+                    name: ""
+                    description: ""
+                    type: Object
+                    optional: true
+                  createTime: *createTime
+              offers: &offers
+                name: ""
+                description: ""
+                type: Object
+                repeated: true
+                optional: true
+                object: &offer
+                  index: &offerIndex
+                    name: ""
+                    description: ""
+                    type: String
+                  price: &price
+                    name: ""
+                    description: ""
+                    type: String
+                  duration: &duration
+                    name: ""
+                    description: ""
+                    type: String
+                  active:
+                    name: ""
+                    description: ""
+                    type: Boolean
+                  createTime: *createTime
+              purchases: &purchases
+                name: ""
+                description: ""
+                type: Object
+                repeated: true
+                optional: true
+                object: &purchase
+                  purchaser: &purchaser
+                    name: ""
+                    description: ""
+                    type: String
+                  expire: &expire
+                    name: ""
+                    description: ""
+                    type: String
+                  createTime: *createTime
+      error: &outputError
+        name: "Error"
+        description: "Output when an error occurs"
+        data:
+          message:
+            name: "Message"
+            description: "The error message"
+            type: String
+  getService:
+    name: ""
+    description: ""
+    inputs:
+      sid: &inputSid
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data: *service
+      error: *outputError
+  getServiceVersion:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      hash: &inputHash
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data: *version
+      error: *outputError
+  serviceExist:
+    name: ""
+    description: ""
+    inputs:
+      sid: &inputSid
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data:
+          exist:
+            name: ""
+            description: ""
+            type: Boolean
+      error: *outputError
+  createService:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      from: &inputFrom
```

Comment:
should we rename into `author`, it makes more sense than `from`

Reply:
it's the `from` parameter for creating / signing the transaction, like all the other tasks that return a transaction.
It's not directly related to the `owner` of the service.

Reply:
I understand that but this api is not related to a transaction on ethereum it's about creating a service. What does it means a from to create a service.
Otherwise we should rename this api with `createServiceTransaction` where the outputs would make more sense and here the from is making sense too.